x-pack/legacy/plugins/maps/public/layers/sources/es_search_source/es_search_source.js

```diff
@@ -367,23 +370,35 @@ export class ESSearchSource extends AbstractESSource {
           defaultMessage: `Results limited to most recent {topHitsSize} documents per entity.`,
           values: { topHitsSize: this._descriptor.topHitsSize }
         });
-        return `${entitiesFoundMsg} ${trimmedMsg}`;
+        return {
+          tooltipContent: `${entitiesFoundMsg} ${trimmedMsg}`,
+          areResultsTrimmed: false
```

Comment:
Should this be set to true? Top hits results are trimmed and show only most recent hits.

Reply:
imho they are not trimmed. It's the intention of the user to limit results in this case.